src/DelimitedDataParser/Exporter.cs

```diff
@@ -195,6 +201,8 @@ public virtual void ExportReader(DbDataReader reader, TextWriter writer, Cancell
                     RenderRow(reader, writer);
 
                     rowIndex++;
+
+                    _progress?.Report(rowIndex);
```

Comment:
We're reporting on each and every line here. Are we comfortable with that, or should we only report on every n lines? We could report on every line for small files and every n lines for large files, bit that's probably overcomplicating things.